CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing guide
+
+**First off all, thank you for taking the time to contribute into Prospero!** The below contents will help you through the steps for getting started with Prospero. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions.
+
+## Documentation
+
+You can find all information and examples in [Prospero documentation](https://docs.wildfly.org/prospero/)
+
+## Development environment
+
+* Latest Maven build tool https://maven.apache.org/
```

Comment:
We should maybe loosen that to say something like "Maven 3.9.x, other versions may work too", so that people don't think they have to download Maven 4, which is gonna be somewhat obscure for some time yet.